### PR TITLE
utils: revive sns hctl repair/rebalance cmds

### PR DESCRIPTION
**Pull request by andriytk**
Revive sns-related `hctl repair/rebalance` commands.

Related to https://jts.seagate.com/browse/EOS-27765.

### CODE REVIEW

**Comment by mssawant** [comment]
@andriytk,  can you please post some mock test results to check if the commands are processed?
Also please squash the commits into one and add more details to the commit please?

**Comment by andriytk** [comment]
@mssawant I didn't want to squash on purpose, I wanted the changes to be linked with the git history. This way it's easier to track which part of the code is really new.

File: rules/sns-repair [hunk 1]
```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
[the remark this answers is not in the document]
Comment (andriytk) [reply rules/sns-repair]: Done.

File: utils/hare-rebalance [hunk 2]
```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```
[the remark this answers is not in the document]
Comment (andriytk) [reply utils/hare-rebalance]: Done.

**Comment by andriytk** [comment]
> ... and add more details to the commit please?

Done.

**Comment by andriytk** [comment]
As for the test results, I do see that the events are added to the events queue at consul kv:

```
$ hctl repair start
2022-02-07 19:13:16,670 [DEBUG] Bound <class 'hax.common.HaxGlobalState'> to an instance <hax.common.HaxGlobalState object at 0x7f735f6dc240>
2022-02-07 19:13:16,672 [DEBUG] Created and configured an injector, config=<function di_configuration at 0x7f735ea9f9d8>
2022-02-07 19:13:16,677 [DEBUG] Starting new HTTP connection (1): 127.0.0.1:8500
2022-02-07 19:13:16,678 [DEBUG] http://127.0.0.1:8500/ "GET /v1/kv/eq-epoch HTTP/1.1" 200 104
2022-02-07 19:13:16,686 [DEBUG] http://127.0.0.1:8500/ "PUT /v1/txn HTTP/1.1" 200 144
2022-02-07 19:13:16,687 [INFO] Written to epoch: 4
$ 
$ hctl repair status
[{"fid": "0x7300000000000001:0xf", "state": 1, "progress": 0}]
$ 
$ consul kv get -recurse eq
eq-epoch:4
eq/2:{"message_type": "device-state-set", "payload": {"node": "centos8", "source_type": "drive", "device": "/dev/loop0", "state": "FAILED"}}
eq/3:{"message_type": "device-state-set", "payload": {"node": "centos8", "source_type": "drive", "device": "/dev/loop0", "state": "FAILED"}}
eq/4:{"message_type": "sns-repair", "payload": {"cmd": "start", "fid": "0x6f00000000000001:0x2f"}}
```

But I'm still to check (maybe debug) whether the events are processed by the RC. So far I didn't see any evidence of it. Any hints or suggestions are very much welcome.

**Review by mssawant — APPROVED** [review]

**Comment by mssawant** [comment]
@andriytk, you can look at `hax/hax/queue/` folder where we have implemented EQ/BQ. We can take this up in separate task if you prefer.

**Comment by andriytk** [comment]
Ok, let's do it in a separate task then...

**Review by vaibhavparatwar — APPROVED** [review]

**Pull request merged**